src/main.cpp

```diff
@@ -18,28 +18,28 @@
 #include "VCF_Constants.h"
 #include "VCF_Tasks.h"
 #include "PedalsSystem.h"
-
-
+#include "VCFEthernetInterface.h"
 
 /* Scheduler setup */
 HT_SCHED::Scheduler& scheduler = HT_SCHED::Scheduler::getInstance();
 
-
-
-/* Ethernet message sockets */ // TODO: Move this into its own interface
-qindesign::network::EthernetUDP protobuf_send_socket;
-qindesign::network::EthernetUDP protobuf_recv_socket;
-
-
-
 void setup() {
-    scheduler.setTimingFunction(micros);
-
-    scheduler.schedule(read_adc1_task);
-    scheduler.schedule(read_adc2_task);
-    scheduler.schedule(buzzer_control_task);
+    std::function<unsigned long()> func = micros;
+    scheduler.setTimingFunction(func);
+
+    //scheduler.schedule(read_adc1_task);
+    //scheduler.schedule(read_adc2_task);
+    //scheduler.schedule(buzzer_control_task);
+    // scheduler.schedule(send_vcf_data_task);
+    // scheduler.schedule(recv_vcr_data_task);
```

Comment:
you can uncomment, but it probably doesn't rly matter since we're going to switch scheduler implementations soon anyway